### PR TITLE
Rework tags

### PR DESCRIPTION
**Pull request by mrthegreensky**
Changed some methods with tags based on suggestions made by a fellow team member.

This new changes contains:
- [x] Addition of new tags when adding tags to claims
- [x] Removing longclick from managing tags

ManageTagsActivity now extends TagAddClaimActivity class. Editing and deleting tags are restricted only to ManageTagsActivity as for future implementation, we will remove the removed or edit the edited tags on the claims themselves. 

As of April 2, 2015 new things added/ to be added:
- [x] Edited tags also edit tags that are currently in claims
- [x] Removed tags also remove the tag from claims.

### CODE REVIEW

File: Team9Project/src/com/indragie/cmput301as1/TagAddToClaimActivity.java [hunk 1]
```diff
@@ -35,6 +36,8 @@
 	// Constants
 	//================================================================================
 	
+	protected static final int ADD_TAG_REQUEST= 1;
+	
 	/**
 	 * Filename for storing tags.
 	 */
```
Comment (williums) [inline Team9Project/src/com/indragie/cmput301as1/TagAddToClaimActivity.java:41]: Formatting is a little bit odd here?

**Comment by williums** [comment]
I saw a little bit of weird formatting with the javadocs and indentation but other than that it looks fine

File: Team9Project/src/com/indragie/cmput301as1/TagAddToClaimActivity.java (hunk 1 above)
Comment (mrthegreensky) [inline Team9Project/src/com/indragie/cmput301as1/TagAddToClaimActivity.java:41]: Thank you. I will fix that up.

**Comment by indragiek** [comment]
Looks good to me!

**Pull request merged**